.github/workflows/build_and_test.yml

```diff
@@ -80,12 +80,14 @@ jobs:
           pyspark=`./dev/is-changed.py -m $pyspark_modules`
           if [[ "${{ github.repository }}" != 'apache/spark' ]]; then
             pandas=$pyspark
+            yarn=`./dev/is-changed.py -m yarn`
```

Comment:
Sorry, I just reviewed `modules.py` and have a small question: if the `common/network-yarn` module has changed, should `yarn` also be set to true?

Reply:
Yes, `source_file_regexes` has `common/network-yarn` and I believe it makes sense.

https://github.com/apache/spark/blob/7c728b2c2d6cbe2b8310cd8cf8aa4c2528b489c8/dev/sparktestsupport/modules.py#L1447-L1460